README.md

```diff
@@ -1,135 +1,180 @@
+### Building Static Files (Deployment)
 
-## Installation
+Note that static files are not committed to the repository - they are built during deployment.
 
-To set up a local instance of [doc.silverstripe.org](https://github.com/silverstripe/doc.silverstripe.org):
+```bash
+# Build and verify locally
+npm run build:docs
 
-* Clone this repository to an empty directory
+# Or for user help
+npm run build:user
 ```
-   git clone https://github.com/silverstripe/doc.silverstripe.org path/to/ssdocs
+
+The test `npm run build` locally
+
+```bash
+# Serve the built static site in the out/ directory
+npm run start
 ```
 
-### Docker install
+### Full Command Reference
 
-No local NodeJS nor gatsby-cli is required for this option.
+```bash
+npm ci                 # Install dependencies (use npm ci, not npm update)
 
-* Make sure docker and docker-compose are installed and docker daemon is running
-* Simply use `./docker/run` to run gatsby commands
-  * `./docker/run build` would be equal to run `gatsby build` within a container
-  * `./docker/run develop -p 8000` would run `gatsby develop -p 8000` within a container.
+# Development with mock data (intended for development without clone, uses same mock data as tests)
+npm run mock           # Dev server with mock data (no cloning needed)
 
-### Native install
+# Clone content
+npm run clone          # Clone content based on DOCS_CONTEXT (default: docs, alias for clone:docs)
+npm run clone:docs     # Clone developer docs in preparation for starting dev or build scripts
+npm run clone:user     # Clone user help docs in preparation for starting dev or build scripts
 
-* Install [Gatsby CLI](https://gatsbyjs.com)
+# Development servers (after cloning)
+npm run dev            # Dev server with developer docs on port 9876  (default: docs context, alias for dev:docs)
+npm run dev:docs       # Dev server with developer docs on port 9876
+npm run dev:user       # Dev server with user help on port 9876
 
-## Developing
+# Build static files for deployment
+npm run build          # Clone content, build static files and copy images (default: docs context, alias for build:docs)
+npm run build:docs     # Clone content, build static files and copy images for developer docs
+npm run build:user     # Clone content, build static files and copy images for user help
 
-Once cloned, from the root of the repository, run the command `yarn dev-docs`
-to instantiate a development server. This will consume all of the markdown files in both major release
-branches and allow you to browse the developer documentation site on `http://localhost:8000` by default
-(see the [Gatsby docs](https://www.gatsbyjs.org/docs/) for instructions on customising the port).
+# Testing
+npm test               # Run all tests
+npm run lint           # Run ESLint (must have 0 errors, 0 warnings)
+npm run start          # Serve built static files from out/ directory
```

Comment:
What does starting the dev server (I assume that's what you mean by serving the files) have to do with testing?